facebook/OpenBIC
# main_v2 - fby3.5: bb: Fix reset problem

fby3.5: bb: Fix reset problem
Summary:
- Fix BB BIC can't reset problem.
The root cause is that Zephyr SDK04 change the wdt of dts architecture.
If we don't enable wdt0 before, the kernel won't find wdt1.

Dependency: #233 

Test plan:
- Build code: Pass
- Warm reset: Pass
- Cold reset: Pass

Log:
1. Check warm reset can do successfully.
- Before fix
[BIC console]
uart:~$ kernel reboot warm
No device named wdt1.
Failed to reboot: spinning endlessly...

- After fix
[BIC console]
uart:~$ kernel reboot warm0)I00:00:00.000,000] <inf> usb_dc_aspeed: select ep[0x81] as IN endpoint
[00:00:00.000,000] <inf> usb_dc_aspeed: select ep[0x82] as IN endpoint
[00:00:00.000,000] <wrn> usb_dc_aspeed: pre-selected ep[0x1] as IN endpoint
[00:00:00.000,000] <wrn> usb_dc_aspee*** Booting Zephyr OS build v00.01.04-3-g32eed3dd510b  ***
Hello, wellcome to yv35 baseboard 2022.1.1
...

2. Check cold reset can do successfully.
- Before fix
[BIC console]
uart:~$ kernel reboot cold
No device named wdt1.
Failed to reboot: spinning endlessly...

- After fix
[BIC console]
uart:~$ kernel reboot cold0)I00:00:00.000,000] <inf> usb_dc_aspeed: select ep[0x81] as IN endpoint
[00:00:00.000,000] <inf> usb_dc_aspeed: select ep[0x82] as IN endpoint
[00:00:00.000,000] <wrn> usb_dc_aspeed: pre-selected ep[0x1] as IN endpoint
[00:00:00.000,000] <wrn> usb_dc_aspee*** Booting Zephyr OS build v00.01.04-3-g32eed3dd510b  ***
Hello, wellcome to yv35 baseboard 2022.1.1
...
